Yoast/wordpress-seo
# Transfer previously used keywords assessment to webworker.

## Summary

This PR can be summarized in the following changelog entry:

* Updates the previously used keywords assessment to be run in the webworker.

## Relevant technical choices:

* The ajax calls are still executed in the window as these aren't render blocking. The result is then send to the webworker using custom messages.

## Test instructions

Should be tested together with:  https://github.com/Yoast/YoastSEO.js/pull/1705

This PR can be tested by following these steps:

* Create a post with a new keyword.
* You should see a green bullet with the text that you've never used this keyword before.
* Save the post.
* Create another post with the same keyword.
* You should see an orange bullet with the text that you've used the keyword once before.
* The link in this text should direct you to your first post.

## Quality assurance


Fixes https://github.com/Yoast/YoastSEO.js/issues/1684
